packages/context/src/binding.ts

```diff
@@ -286,7 +286,9 @@ export class Binding {
   }
 
   /**
-   * Bind the key to a BindingProvider
+   * Bind the key to a Provider class
+   *
+   * @param provider - the class constructor of a value provider
    */
```

Comment:
Can you please add an small code example showing a custom Provider class and how it is bound?